index.js

```diff
@@ -59,7 +59,7 @@ class ClinicDoctor {
 
     proc.once('exit', function (code, signal) {
       // Windows uncaught SIGINT has exit code 3221225786
-      /* istanbul ignore if: windows hack */
+      /* istanbul ignore next: windows hack */
```

Comment:
`if` didn't work? See https://github.com/gotwarlost/istanbul/blob/master/ignoring-code-for-coverage.md#ignore-an-if-path

Reply:
No, it didn't - https://circleci.com/gh/nearform/node-clinic-doctor/1111?utm_campaign=vcs-integration-link&utm_medium=referral&utm_source=github-build-link - and we've been trying to figure out why not. Formatting of the if block doesn't seem to make any difference. As expected `else` doesn't skip it, and `if|else` doesn't either. `next` works fine, and Kamil verified it doesn't affect subsequent code. For future reference, I'd like to understand why `if` didn't behave as expected.